packages/renderer/MdToHtml/rules/tableHorizontallyScrollable.ts

```diff
@@ -0,0 +1,19 @@
+export default {
+	plugin: (markdownIt: any) => {
+		markdownIt.renderer.rules.table_open = function(tokens: any[], idx: number, options: any, _env: any, self: any) {
```

Comment:
Please don't use `any` for any of the variables that you use. So tokens, options and self should have a type. You can get it from the markdown-it package

Reply:
this code is copy from packages/renderer/MdToHtml/rules/source_map.ts.
The exact type of `self` variable is Renderer, but markdown-it has not been exported.

Reply:
change to 
```
markdownIt.renderer.rules.table_open = function(tokens: MarkdownIt.Token[], idx: number, options: MarkdownIt.Options, _env: any, renderer: any) {
```

Reply:
@laurent22   Need I rebuild `packages/renderer/MdToHtml/rules/{checkbox.ts, code_inline.ts, fence.ts, fountain.ts, highlight_keywords.ts, html_image.ts, image.ts, katex.ts, link_close.ts, link_open.ts, mermaid.ts, sanitize_html.ts, source_map.ts}` to add tokens, options types?